app/src/modules/settings/routes/extensions/extensions.vue

```diff
@@ -103,3 +149,4 @@ fetchExtensions();
 	margin-top: 24px;
 }
 </style>
+../../../../composables/use-reload-guard
```

Comment:
![vscode-again](https://github.com/directus/directus/assets/5363448/52a42531-9bf1-49c7-95ef-f53d2ba02e4f)

_\~ Best wishes, VS Code_

```suggestion
```

Reply:
Hahaha damn you vscode, really need to stop smacking that "update imports" button when moving files